java/client/src/main/java/glide/api/commands/BitmapBaseCommands.java

```diff
@@ -0,0 +1,64 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.commands;
+
+import glide.api.models.commands.BitmapOptions;
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * Supports commands and transactions for the "Bitmap Commands" group for standalone and cluster
+ * clients.
+ *
+ * @see <a href="https://redis.io/docs/latest/commands/?group=bitmap">Bitmap Commands</a>
+ */
+public interface BitmapBaseCommands {
+    /**
+     * Counts the number of set bits (population counting) in a string stored at <code>key</code>.
+     *
+     * @see <a href="https://redis.io/commands/bitcount/">redis.io</a> for details.
```

Comment:
In some of my other PRs Sho has asked me to switch to using valkey docs, should we change the links in all these Javadocs to valkey?
Note however that the link above the interface definition does not have an equivalent valkey link since valkey doesn't have links for specific command groups yet
```suggestion
     * @see <a href="https://valkey.io/commands/bitcount/">valkey.io</a> for details.
```

Reply:
@acarbonetto mentioned that we would do a follow up PR to update all the links because because there are some missing links (for example, groups). @acarbonetto should we reconsider this?

Reply:
Okay good to know, yeah just thought I'd point it out. If we want to keep them as redis.io and update later I'm fine with that

Reply:
Either one is fine.  if you don't update this now, we will need to do it as part of the mass update. 
The groups links aren't available yet, so we are waiting valkey documentation before doing the update.